MMM-pages.js

```diff
@@ -113,27 +118,24 @@ Module.register('MMM-pages', {
         this.sendNotification('PAGE_NUMBER_IS', this.curPage);
         break;
       case 'PAUSE_ROTATION':
-        if (!this.rotationPaused) {
-          Log.log('[Pages]: pausing rotation due to notification');
-          clearInterval(this.timer);
-          clearInterval(this.delayTimer);
-          this.rotationPaused = true;
-        } else {
-          Log.warn('[Pages]: Was asked to paused but rotation was already paused!');
-        }
+        this.setRotation(true);
         break;
       case 'RESUME_ROTATION':
-        if (this.rotationPaused) {
-          Log.log('[Pages]: resuming rotation due to notification');
-          this.resetTimerWithDelay(this.rotationDelay);
-          this.rotationPaused = false;
-        } else {
-          Log.warn('[Pages]: Was asked to resume but rotation was not paused!');
-        }
+        this.setRotation(false);
```

Comment:
It feels kinda weird that resuming rotation sets rotation to false, and that pausing rotation sets rotation to true... can we invert the logic in `setRotation`?

Reply:
You are totally right. With renaming the function the reverse logic makes more sense.